package-lock.json

```diff
@@ -7,18 +7,18 @@
     "@types/crypto-js": {
       "version": "3.1.38",
       "resolved": "https://registry.npmjs.org/@types/crypto-js/-/crypto-js-3.1.38.tgz",
-      "integrity": "sha512-cdMO/lU4rX1dJqYJERGuMIrNMoizPFXIXMPuhcxXq+4m36ZEUQ0E/3t0mnvbbfLQXuas6BFY8uoRs19s/m+XXg==",
+      "integrity": "sha1-4TZ/dz7d4phrqa6+478c3z6Bil0=",
```

Comment:
https://github.com/npm/npm/issues/17749
https://github.com/OpenZeppelin/zeppelin-solidity/pull/648

This seems to be related to this issue in npm; different environments may update integrity. I did not revert as there are some added entries that are needed; but this can be reverted & rebuilt to spec.

Reply:
stackoverflow fix for this: 
https://stackoverflow.com/questions/47638381/what-did-package-lock-json-change-the-integrity-hash-from-sha1-to-sha512

Reply:
went through steps from above SO post; generated lock file is the same as in this commit.